### PR TITLE
Add RPC Rate Limits and Re-Add OnFinality Endpoints

### PR DESCRIPTION
**Pull request by themacexpert**
### Description

Adds RPC Rate Limits and Re-Adds OnFinality Endpoints

### Checklist

- [x] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: .snippets/text/builders/get-started/endpoints/moonriver.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-    | POKT Network | <pre>```https://moonriver-mainnet.gateway.pokt.network/v1/lb/62a74fdb123e6f003963642f```</pre> |
-    |  UnitedBloc  |                       <pre>```https://moonriver.unitedbloc.com```</pre>                        |
-
+      |   Provider   |                                            RPC URL                                             |   Limits   |
+      |:------------:|:----------------------------------------------------------------------------------------------:|:----------:|
+      |    Blast     |                     <pre>```https://moonriver.public.blastapi.io```</pre>                      | 80 req/sec |
+      |   Dwellir    |                       <pre>```https://moonriver-rpc.dwellir.com```</pre>                       | 20 req/sec |
+      |  OnFinality  |                  <pre>```https://moonriver.api.onfinality.io/public```</pre>                   | 40 req/sec |
+      | POKT Network | <pre>```https://moonriver-mainnet.gateway.pokt.network/v1/lb/62a74fdb123e6f003963642f```</pre> | 25 req/sec |
+      |  UnitedBloc  |                       <pre>```https://moonriver.unitedbloc.com```</pre>                        | 32 req/sec |
```
Comment (eshaben) [inline .snippets/text/builders/get-started/endpoints/moonriver.md:8]:
```suggestion
      |  UnitedBloc  |                       <pre>```https://moonriver.unitedbloc.com```</pre>                        | 32 req/sec |

```

To preserve spacing between elements

**Review by eshaben — COMMENTED** [review]
ohh boyyyy onfinality is back!? looks good!

**Comment by themacexpert** [comment]
@eshaben Yup it's back! I'm waiting to hear back from Opslayer on Our Moonbeam endpoint rate limits, then will be ready to merge

**Comment by themacexpert** [comment]
@eshaben This is ready to go now!

**Review by eshaben — APPROVED** [review]
LGTM! Thanks!!

**Pull request merged**